src/modules/LanUdevAuto.rb

```diff
@@ -194,39 +198,43 @@ def Import(settings)
     end
 
     def Write
-      comment = [
-        "# Generated by autoyast",
-        "# program run by the persistent-net-generator.rules rules file.",
-        "#",
-        "# You can modify it, as long as you keep each rule on a single line."
-      ]
+      template = "SUBSYSTEM==\"net\", ACTION==\"add\", DRIVERS==\"?*\", %s==\"%s\", NAME=\"%s\""
+
       rules = []
-      Builtins.foreach(@udev_rules) do |rule|
-        rules = Builtins.add(
-          rules,
-          Builtins.sformat(
-            "SUBSYSTEM==\"net\", ACTION==\"add\", DRIVERS==\"?*\", %1==\"%2\", NAME=\"%3\"",
-            Ops.get_string(rule, "rule", ""),
-            Ops.get_string(rule, "value", ""),
-            Ops.get_string(rule, "name", "")
-          )
-        )
+
+      @udev_rules.each do |rule|
+        opt = rule["rule"] || ""
+        value = rule["value"] || ""
+        devname = rule["name"] || ""
+
+        rules << template % [opt, value.downcase, devname]
       end
-      if Ops.greater_than(Builtins.size(rules), 0)
-        Builtins.y2milestone(
-          "Writing AY udev rules for network (will replace original rules from 1st stage)"
-        )
-        if AllowUdevModify()
-          SCR.Write(path(".udev_persistent.rules_comment"), comment)
-          SCR.Write(path(".udev_persistent.rules"), rules)
-        end
+
+      if !rules.empty? && AllowUdevModify()
+        SetAllLinksDown()
```

Comment:
This is new, introduced in 8561a6f08971e236dbec1a5d2eaceb01486d7d68 named "Wait until udev settles down when updating udev rules."
It seems it is a great opportunity to break things. Why is it needed?